CMakeLists.txt

```diff
@@ -22,16 +22,15 @@ find_package(Boost COMPONENTS program_options unit_test_framework REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
 ## Dhrystone
-add_custom_command(OUTPUT dhry/dhry_1.o dhry/dhry_2.o
-                   COMMAND make
-                   WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/dhry
-)
+# add_custom_command(OUTPUT dhry/dhry_1.o dhry/dhry_2.o
```

Comment:
Please remove commented code, if the purpose is to keep it around to use it in special context, we can add a flag or target for it.